pulp_rpm/app/models/repository.py

```diff
@@ -233,7 +233,7 @@ class RpmRepository(Repository, AutoAddObjPermsMixin):
         RpmPackageSigningService, on_delete=models.SET_NULL, null=True
     )
     package_signing_fingerprint = models.TextField(null=True, max_length=40)
-    original_checksum_types = models.JSONField(default=dict)
+    original_checksum_types = models.JSONField(default=dict)  # DEPRECATED, remove in 3.27+
```

Comment:
```suggestion
    original_checksum_types = models.JSONField(default=dict)  # DEPRECATED, remove in 3.29+
```

Reply:
Since the signature of the `publish()` task entry point is being changed, it'll have to be tweaked over the course of a few releases if we want to maintain hot-upgradability.